boards/arm/nrf52840_mdk/nrf52840_mdk.dts

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright (c) 2017 Linaro Limited
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+#include <nordic/nrf52840.dtsi>
```

Comment:
change to include <nordic/nrf52840_qiaa.dts>

Reply:
pk

Reply:
ok